src/components/common/Chip/index.tsx

```diff
@@ -4,13 +4,14 @@ import CancelDefault from '@/assets/icons/cancel-default.svg?react';
 
 interface ChipProps {
   tag: string;
+  onClick: () => void;
 }
 
-const Chip = ({ tag }: ChipProps) => {
+const Chip = ({ tag, onClick }: ChipProps) => {
   return (
     <Wrapper>
       <span># {tag}</span>
-      <CancelDefault style={{ width: '1.5rem', height: '1.5rem' }} />
+      <CancelIcon onClick={onClick} />
```

Comment:
온클릭 핸들러가 있으며 버튼이라는 건데 
버튼은 html 태그를 button으로 하시는 게 좋아요

CancelIcon은 svg 아이콘에 스타일링만 더해서 쓰고 있어서 
차라리 
```suggestion
<CancelIconButton>
  <CancelDefault/>
</CancelIconButton>

const CancelIconButton = styled.button`
...
```
이런 식으로 감싸는 버튼 태그를 만드는 게 좋아보입니다.